sigma/backends/loki/loki.py

```diff
@@ -297,14 +297,14 @@ def set_expression_templates(negated: bool) -> None:
 
     # Loki-specific functionality
     add_line_filters: bool = False
-    case_insensitive: bool = True
+    case_sensitive: bool = True

```

Comment:
Case insensitive being true means case sensitive being false.
```suggestion
    case_sensitive: bool = False

```